lib/Backends/NNPI/InferenceContext.cpp

```diff
@@ -462,7 +462,7 @@ void InferenceContext::execute(RunIdentifierTy runId,
         // Then query all errors using another wait call (should return
         // immediately).
         NNPIInferenceErrorCode tmpRes =
-            nnpiCommandListWait(commandList_, deviceOptions_->inferTimeout,
+            nnpiCommandListWait(commandList_, deviceOptions_->inferTimeoutMs,
```

Comment:
@omromano : are we sure about this change? nnpiCommandListWait arguments suggest it expect a timeoutUs.

Reply:
@arunm-git you are right, I will fix it.
Note that now HostManager::testTimeout test will now fail because flags::NNPITimeout has changed to flags::NNPITimeoutMs (and the test is setting it to 1 which is too high).
If a better granularity is needed, we should consider changing this flag to Us instead of Ms.